### ISSUE TITLE
`?` is omitted when printing

### ISSUE THREAD

**Issue by stackmystack** [fazibear/colorize#98]
I think this is a recent, but a really bad bug:

```
irb(main):001:0> puts "?".red

=> nil
irb(main):002:0> puts "?.".red
.
=> nil
irb(main):003:0> puts "!.".red
!.
=> nil
irb(main):004:0>
```

**Comment by stackmystack** [comment]
Continued:

```
irb(main):004:0> puts "!.".red.inspect
"\x01\e[0;31;49m\x02!.\x01\e[0m\x02"
=> nil
irb(main):005:0> puts "?".red.inspect
""
=> nil
```

**Comment by fazibear** [comment]
Yep! Regex fixed! `v1.0.3` released!

**Comment by stackmystack** [comment]
Thanks, I can confirm, it works.
